app/views/layouts/_policy_sim.html.haml

```diff
@@ -1,5 +1,8 @@
 - url = url_for_only_path(:action => 'policy_sim_add')
 #main_div
+  - if @right_cell_text && !@explorer
+    %h1
+      = URI.unescape(@right_cell_text)
```

Comment:
Why unescape?

Reply:
Thank you for this good question, David. I was inspired here https://github.com/hstastna/manageiq-ui-classic/blob/master/app/views/layouts/_content.html.haml#L28 however I was not sure if we really want this. Anyway, it is a place where we are if displaying title works well (explorer screens and (not just) policy simulation page). Personally, I think that it is not necessary to add unescape (my change) for now, but tomorrow.. ? Depends on what can be in @right_cell_text tomorrow. Also, another thing: I am not sure if it is not obsolete already. So.. any advice? Should I remove it?